compiler/src/main/java/org/qbicc/context/CompilationContext.java

```diff
@@ -80,6 +80,25 @@ default ClassContext getClassContextForLoader(VmClassLoader classLoaderObject) {
      */
     void registerEntryPoint(ExecutableElement method);
 
+    /**
+     * InvokableElements are executable elements methods that must
+     * be included in the final image because they have either been
+     * entered into a dispatching table and thus may be invoked via
+     * an indirect call through a function pointer.
+     * These elements are not necessarily EntryPoints, but the program
+     * points that invoke them have been lowered to the point where it
+     * is no longer possible to determine they are reachable from the graph.
+     *
+     * @param element the element to register as invokable.
+     */
+    void registerInvokableElement(ExecutableElement element);
+
+    default void registerRuntimeFunction(ExecutableElement element) {
+        registerInvokableElement(element);
+    }
```

Comment:
What is this one for?  Is there a possibility that this would have different semantics from the other one?